typescript/cli/src/config/warp.ts

```diff
@@ -151,9 +151,28 @@ export async function createWarpRouteDeployConfig({
     const isNft =
       type === TokenType.syntheticUri || type === TokenType.collateralUri;
 
-    const interchainSecurityModule = advanced
-      ? await createAdvancedIsmConfig(context)
-      : createDefaultWarpIsmConfig(owner);
+    /**
+     * The logic from the cli is as follows:
+     *  --advanced flag is provided: the user will have to build their own configuration using the available ISM types
+     *  --yes flag is provided: the default ISM config will be used (Trusted ISM + Default fallback ISM)
+     *  -- no flag is provided: the user must choose if the default ISM config should be used:
+     *    - yes: the default ISM config will be used (Trusted ISM + Default fallback ISM)
+     *    - no: the default fallback ISM will be used
+     */
+    let interchainSecurityModule: IsmConfig;
+    if (advanced) {
+      interchainSecurityModule = await createAdvancedIsmConfig(context);
+    } else if (context.skipConfirmation) {
+      interchainSecurityModule = createDefaultWarpIsmConfig(owner);
+    } else if (
```

Comment:
From a UX experience, i prefer all token related prompts to be together. In this case, a user selects their type, but then gets asked about ISMs, and then goes back to configuring the token. An example of this is for collateralVault
![Screenshot 2024-09-25 at 11 52 32 AM](https://github.com/user-attachments/assets/ed3c137a-5c10-4cd7-9755-93c1465389ec)

Reply:
i suggest moving the type selection to above the switch statement. but overall, good progress

Reply:
Agreed it makes more sense to have it there, thank you for the suggestion 🔥